cmd/kind/load/docker-image/docker-image.go

```diff
@@ -88,18 +96,35 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 
 	// pick only the user selected nodes and ensure they exist
 	// the default is all nodes unless flags.Nodes is set
-	selectedNodes := nodes
+	candidateNodes := nodes
 	if len(flags.Nodes) > 0 {
-		selectedNodes := []clusternodes.Node{}
+		candidateNodes := []clusternodes.Node{}
```

Comment:
this has the same bug as the old code, this should be `=` not `:=`

Reply:
done